draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -627,6 +627,28 @@ in cache before performing any followup queries.
 With these optimizations in place, and conforming DNS servers,
 using SVCB does not add network latency to connection setup.
 
+Moreover, if an A or AAAA response arrives before an SVCB response, the
+client SHOULD wait up to CD milliseconds before connecting as if the
+SVCB query returned NODATA, but MUST NOT transmit any information that
+could be altered by an SVCB response until the SVCB response arrives.
+For example, a TLS connection may be altered by the "esnikeys" value
+of an SVCB response. CD (Connection Delay) is a configurable parameter.
+The recommended value is 50 milliseconds, as per the guidance in {{!HappyEyeballsV2=RFC8305}}.
+
+If an SVCB record is consistent with an active or in-progress connection,
+the client may continue using that connection using any information provided
+by the SVCB RR. For example, if an SVCB record with a "esnikeys" value and
```

Comment:
I think changing between "RR" and "record" in this sentence is slightly confusing.

Reply:
I'll stick with record!